### DISCUSSION TITLE
Display sonar while crewing a periscope

### DISCUSSION THREAD

**Discussion by Man0fGlory** [FakeFishGames/Barotrauma#9526] [Ideas and suggestions]
Situational awareness is important while crewing periscopes, and adding the ability to view sonar by the operator would be helpful. This could be done through a purchaseable upgrade or wearable item.
Without:
![Without](https://user-images.githubusercontent.com/75109541/177161665-43fe3f3a-ddf2-456a-a417-d3b376addf99.jpg)
With:
![With](https://user-images.githubusercontent.com/75109541/177161677-65c945d1-7181-4c00-b214-929227684997.jpg)

Thank you for your attention

**Comment by mygamingaccount** [comment]
This is entirely unnecessary, you can get the contact bearing ahead of time if you listen to the captain

**Reply by Man0fGlory** [reply]
If what you are saying was the norm and not the exception, this proposal would not exist.

**Reply by mygamingaccount** [reply]
Find yourself a better crew

**Reply by Man0fGlory** [reply]
Since we are giving "productive" feedback, maybe I should play a different game also.

**Reply by mygamingaccount** [reply]
Excuse me?!

**Reply by Man0fGlory** [reply]
Well, if you're challenging my proposal because you argue that it is unnecessary because players are just bad, it's only fair that I get to defend it using the same type of "constructive" argument.

**Reply by mygamingaccount** [reply]
The SONAR display on your periscope won't improve the gameplay for you. It's never going to be anywhere near the same as having a proper crew that communicates.
If you're joining a random lobby, test if your helmsman can talk. If not, quit and move to another ship.

No reason to pretend that you're an offended primadonna for people calling out your suggestion. This change would make the game worse, not better.

**Reply by Man0fGlory** [reply]
Making assumptions whenever a feature will improve the gameplay for someone arguing the opposite is a bit presumptuous, don't you think? Also, discriminating against people who choose to play without a microphone is a sound policy for a community... Testing people if they have a microphone and promptly leaving is just as rude as leaving them behind as community members. This is not my way of doing things, and that is why I try, at least, since it is within my power to propose changes in order to attempt to improve things.

Accusing me of being an "offended primadonna" for highlighting a cheap way of arguing is just immature. You are, of course, entitled to your own opinion, which should be respected in the same way as mine, btw. Now I would ask you, though, to refrain from calling me names in order to get some emotional response from me or attempting to derail this discussion into a personal quarrel. I won't dignify future attempts at argumentum ad personam with a response.

**Reply by MateusJK15** [reply]
> Find yourself a better crew

What about singleplayer ? The AI won't keep updating you about the bearing of every single enemy all the time.

**Comment by TopDogRolly** [comment]
QoL stuff always welcome

**Comment by Hazmat616** [comment]
Would be nice if it’s at least the sonar of the guns field of view. Or even have the ability to release a directional ping from a hardpoint or spotlight itself.
